pkgs/applications/science/chemistry/gwyddion/default.nix

```diff
@@ -45,7 +45,7 @@ stdenv.mkDerivation rec {
   
   buildInputs = with stdenv.lib;
     [ gtk2 fftw ] ++
-    optionals openglSupport [gnome2.gtkglext] ++
+    optional (openglSupport && (!stdenv.hostPlatform.isDarwin)) gnome2.gtkglext ++
```

Comment:
Why not instead change `openglSupport ? true` to `openglSupport ? !stdenv.isDarwin`.

Reply:
Because I hadn't thought of that... thank you.